CHANGELOG.md

```diff
@@ -265,132 +101,49 @@ If you made modifications to these files in your project, you will have to manua
 
 ### New template files
 
-The following template files have been added in 12.0.0:
+The following template files have been added in 12.1.0:
```

Comment:
Think these were added in `12.0.0` so they can be removed from this section for `12.1.0`.

Reply:
Cool. yeah, they were. was leaving them there until I checked. Maybe no new templates for 12.1.0?